KPI-team1-frontend/src/model/kpi.ts

```diff
@@ -0,0 +1,44 @@
+enum KpiPeriodicity {
+  Daily = "daily",
+  Weekly = "weekly",
+  Monthly = "monthly",
+  Quarterly = "quarterly",
+  Yearly = "yearly",
+}
+
+export interface Kpi {
+  kpi_id: number;
+  kpi_name: string;
+  description: string | null;
+  periodicity: KpiPeriodicity;
+  value_min: Number | null;
+  value_max: Number | null;
+  unit: string | null;
```

Comment:
I think those shouldn't be null :P But as they are now in the DB let's leave it like that